hedera-node/configuration/testnet/upgrade/feeSchedules.json

```diff
@@ -409,6 +409,51 @@
                 }
             } ]
         }
+    }, {
+        "transactionFeeSchedule": {
+            "hederaFunctionality": "TokenAirdrop",
+            "fees": [
+                {
+                    "subType": "DEFAULT",
```

Comment:
These feeSchedules should be not `DEFAULT` from `CryptoTransfer`

Reply:
After the merge with develop, there is no changes in these files